packages/graphql/src/toESModules.js

```diff
@@ -0,0 +1,42 @@
+import { EOL } from 'os';
+
+export default function(source) {
+  if (typeof source !== 'string') {
+    return source;
+  }
+
+  return replaceRequires(replaceModuleExports(source));
+}
+
+function replaceModuleExports(source) {
+  return source
+    .replace('module.exports = doc', 'export default doc')
+    .replace(
+      /module\.exports\["(.*)"] = oneQuery\(doc, "(.*)"\)/g,
```

Comment:
We should have some comment explaining what the regex is doing.

Reply:
I think the name of the function `replaceModuleExports` already does the job. What do you think?

Reply:
Fair enough :+1: